### PR TITLE
revert some changes, could be buggy

### PR DESCRIPTION
**Pull request by TriplEight**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -99,7 +99,7 @@ build-windows:
 publish-docker:
   stage:                           publish
   only:                            *releaseable_branches
-  <<:                              *no_git
+  # <<:                              *no_git
```
Comment (soc1c) [inline .gitlab-ci.yml]: please remove it or not

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -140,23 +140,6 @@ publish-onchain:
   tags:
     - linux-docker
 
-# configures aws for fast uploads/syncs
-.s3-before-script:                &s3-before-script
```
Comment (soc1c) [inline .gitlab-ci.yml]: why is this being removed?

File: .gitlab-ci.yml (hunk 1 above)
Comment (General-Beck) [reply .gitlab-ci.yml]: remove

File: .gitlab-ci.yml (hunk 2 above)
Comment (General-Beck) [reply .gitlab-ci.yml]: moved to docker parity/awscli @TriplEight ?

Comment (gabreal) [reply .gitlab-ci.yml]: it's [here](https://github.com/paritytech/scripts/blob/master/awscli/config) now

**Pull request merged**